actions/release/reset-draft/entrypoint/main.go

```diff
@@ -69,14 +71,35 @@ func main() {
 		return
 	}
 
+	releaseToDelete := releases[0]
```

Comment:
Should we also remove the section below:

```

	if !releases[0].Draft {
		fmt.Println("Latest release is published, exiting.")
		return
	}
```

I think this logic is covered by both branches in the `for` loop you added.

Reply:
Unfortunately, the `for` loop case I added only applies to when we pass in `config.Version`. In the case where no version is passed in we want to perform the `releases[0].Draft` check still. Maybe it would be clearer if I added that case as an `else if`?

Reply:
Ah, gotcha. I think this logic makes sense. It might be slightly easier to read as follows:

```
var releaseToDelete github.Release // or whatever the type actually is

if config.Version == "" {
  releaseToDelete = releases[0]
} else {
  // existing for loop that eventually sets releaseToDelete = r
}

fmt.Printf("found draft with version: '%s', deleting", releaseToDelete.Version)
```

But I don't feel strongly either way.

Reply:
Fixed per your suggestion - I prefer that!